### ISSUE TITLE
Should normalize update minor_version?

### ISSUE THREAD

**Issue by vidartf** [jupyter/nbformat#328]
Currently if I run a v 4.0 notebook through `nbformat.validator.normalize`, it will add cell ids, but it will leave the minor version as 0. Shouldn't it be updated to 5 to highlight that it now includes some fields that was added in that version of the schema? Since the previous versions have the schema with `additionalProperties: false`, the new notebook will be invalid against the 4.0 schema.

**Comment by blink1073** [comment]
Yes, it makes sense to me that whenever a normalize occurs, it should bump to the version that was used to normalize.
